packages/banner/src/presenters/IconBackground/icon-background.scss

```diff
@@ -0,0 +1,51 @@
+@import "../../tokens.scss";
+
+.hig__banner__icon-background {
+  display: flex;
+  flex-grow: 0;
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: center;
+  width: $banner-wrapper-min-height;
+  margin: 0;
+
+  svg * {
+    fill: color(hig-white);
+  }
+}
+
+.hig--light-theme {
+  .hig__banner--urgent &.hig__banner__icon-background {
```

Comment:
One really important thing to notice here. **Each element that has theme variations must have the theme class applied to it directly.**

I'm trying to prevent theme values from cascading.